### DISCUSSION TITLE
Blog Tracker

### DISCUSSION THREAD

**Discussion by aceofspaders** [AprilSylph/XKit-Rewritten#289] [Suggestion Box]
Hi!
First of all, thank you so much for the hard work you've done and continue to do to make the Tumblr environment more friendly for users! It's so very appreciated)
Now to the point of my message: I've read "Blog Tracker" section on the page about the features you're not gonna include in XKit Rewritten, and **yet** I'd like to ask you to consider adding it still, because this feature was very handy - which is why I loved it so much and was so very sad when it was broken - and neither push notifications nor Blog Subscriptions Timeline do quite the same thing that Blog Tracker did. Besides, BS Timeline is very hard to find and, for some reason, there are blogs you can't subscribe to while Blog Tracker let you track any and all blogs.
Thank you in advance for taking the time to read this,
Di

**Comment by AprilSylph** [comment]
I agree the Blog Subscriptions timeline is hard to find, but that's why I linked to it on the wiki. Since there's no button to access it from within the Tumblr dashboard, I suggest bookmarking it if you intend to use it.

> for some reason, there are blogs you can't subscribe to

Tumblr only allows you to subscribe to blogs which you're following.

**Reply by aceofspaders** [reply]
Thank you for replying so quickly and for your suggestion! 

I've already bookmarked the Blog Subscriptions timeline but finding a way to easily access it doesn't change the fact that its functions are not the same as Blog Tracker's were nor the fact that Blog Tracker is far handier.

>Tumblr only allows you to subscribe to blogs which you're following.

Yes, I'm well aware of that. But the problem is that it doesn't allow me to subscribe even to some of my mutuals. 
But that's beside the point.

So, just to clear up any possible confusion, are you going to consider adding Blog Tracker still, considering there's at least one person - and, I suspect, there are other people who would like it if it returned - requesting it?

Thank you for taking the time to read and reply to this again,
Di

**Reply by AprilSylph** [reply]
No. I would if Tumblr provides a method for opening peepr without reloading the page, like the old dashboard had, I might consider it. But requesting such a feature from Tumblr when there exist *two* working alternatives to this feature is not reasonable, especially when there are still other features to be built without the need for Tumblr's involvement.

I suggest contacting Tumblr support about the blogs you're unable to subscribe to.
